File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function (nums1, nums2) {
    const newArray = nums1.concat(nums2).sort((a, b) => a - b);
    console.log(newArray);
    if (newArray.length % 2 === 1) {
        console.log(newArray);
        return (newArray[Math.floor(newArray.length / 2)]);
    } else {
        console.log(newArray);
        return (newArray[newArray.length / 2] + newArray[newArray.length / 2 - 1]) / 2;
    }
};

File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.rb
# @param {Integer[]} nums1
# @param {Integer[]} nums2
# @return {Float}
def find_median_sorted_arrays(nums1, nums2)
    nums1.concat(nums2)
    nums1.sort!
    return nums1[nums1.size/2] if nums1.size % 2 == 1
    value = (nums1[(nums1.size/2).ceil] + nums1[(nums1.size/2).floor - 1])
    value.to_f / 2
end